kork/kork-web/src/main/groovy/com/netflix/spinnaker/config/OkHttp3ClientConfiguration.groovy

```diff
@@ -83,36 +83,14 @@ class OkHttp3ClientConfiguration {
 
   public OkHttp3ClientConfiguration(OkHttpClientConfigurationProperties okHttpClientConfigurationProperties,
                                     OkHttp3MetricsInterceptor okHttp3MetricsInterceptor) {
-    this(okHttpClientConfigurationProperties, okHttp3MetricsInterceptor, null, null, null,
+    this(okHttpClientConfigurationProperties, okHttp3MetricsInterceptor, HttpLoggingInterceptor.Level.BASIC, null, null,
```

Comment:
Not sure what's happening in the code before this PR, but wouldn't we be better off adding an HttpLoggingInterceptor.Level constructor argument instead of hard-coding this?

Reply:
Adding constructor argument means changing the signature of the two constructors and the corresponding callers. That will be a bigger change. Do you want me to proceed with that? Even if that is the case, I feel it can be done in a different PR in future.

Reply:
It must be retrofit1's usage which didn't need retrofit2's log level hence was null.

Reply:
If only retrofit1 used the constructor, can we remove it altogether?

Reply:
The code previously used the constructor for retrofit1 client is now getting retrofit2 client and hence having null for log level fails while creating the new client.

Reply:
I will try to add the log level constructor parameter.